src/pages/new/index.js

```diff
@@ -53,6 +55,46 @@ export default () => (
         { text: "Work with us", onClick: () => {} },
       ]}
     />
+    <TwoParagraphBlock
```

Comment:
Actually, you shouldn't add this to the home page because this isn't actually on the home page. It should be on the Volunteer page, but my PR that adds the Volunteer page hasn't merged yet: https://github.com/ShelterTechSF/sheltertech.org/pull/80

Reply:
Sorry, I just realized that. I'll remove this component from the home page.